scripts/libnghttp2/1.26.0/script.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+MASON_NAME=libnghttp2
+MASON_VERSION=1.26.0
+MASON_LIB_FILE=lib/libnghttp2.a
+MASON_PKGCONFIG_FILE=lib/pkgconfig/libnghttp2.pc
+
+. ${MASON_DIR}/mason.sh
+
+
+function mason_load_source {
+    mason_download \
+        https://github.com/nghttp2/nghttp2/releases/download/v1.26.0/nghttp2-1.26.0.tar.gz \
```

Comment:
Minor: please use `${MASON_VERSION}` in place of these hardcoded version numbers.